### PR TITLE
Updates to built-in slash command functionality

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: 
- https://github.com/mattermost/mattermost-webapp/pull/10957
- https://mattermost.atlassian.net/browse/MM-46468

Note: This is a release v7.3 update.

### CODE REVIEW

File: site/content/integrate/slash-commands/built-in/_index.md [hunk 1]
```diff
@@ -35,7 +35,8 @@ Each Mattermost installation comes with some built-in slash commands that are re
...
 | /shrug *{message}*                                    | Add `¯\_(ツ)_/¯` to your message                                                                   | /shrug oh well                                             |
 | /status *{emoji_name}* *{descriptive status_message}* | Set a custom status that includes an optional emoji and a descriptive status message              | /status sick Feeling unwell and taking time off to recover |
-| /status clear                                         | Clear the current status                                                                          | /status clear                                              |
+| /status clear                                         | Clear the current status                                                                          | /status clear
+| /marketplace                                          | Open the Mattermost Marketplace model                                                             | /marketplace
```
Comment (neflyte) [inline site/content/integrate/slash-commands/built-in/_index.md:39]: I'm not sure what "Mattermost Marketplace model" is referring to. Does this mean the in-product Marketplace?

**Review by neflyte — CHANGES_REQUESTED** [review]

File: site/content/integrate/slash-commands/built-in/_index.md (hunk 1 above)
Comment (cwarnermm) [reply site/content/integrate/slash-commands/built-in/_index.md:39]: It does indeed! Great catch! 
@justinegeffen In the Developer documentation, we've been distinguishing between the 2 marketplaces by "in-product" and "web". Any concerns with this approach from your perspective?

Comment (cwarnermm) [inline site/content/integrate/slash-commands/built-in/_index.md:39]:
```suggestion
| /marketplace                                          | Open the Mattermost product Marketplace                                                             | /marketplace
```

File: site/content/integrate/slash-commands/built-in/_index.md [hunk 2]
```diff
@@ -35,7 +35,8 @@ Each Mattermost installation comes with some built-in slash commands that are re
...
 | /shrug *{message}*                                    | Add `¯\_(ツ)_/¯` to your message                                                                   | /shrug oh well                                             |
 | /status *{emoji_name}* *{descriptive status_message}* | Set a custom status that includes an optional emoji and a descriptive status message              | /status sick Feeling unwell and taking time off to recover |
-| /status clear                                         | Clear the current status                                                                          | /status clear                                              |
+| /status clear                                         | Clear the current status                                                                          | /status clear
+| /marketplace                                          | Open the Mattermost product Marketplace                                                             | /marketplace
```
Comment (cwarnermm) [inline site/content/integrate/slash-commands/built-in/_index.md:39]:
```suggestion
| /marketplace                                          | Open the Mattermost product Marketplace                                                           | /marketplace
```

File: site/content/integrate/slash-commands/built-in/_index.md [hunk 3]
```diff
@@ -35,7 +35,8 @@ Each Mattermost installation comes with some built-in slash commands that are re
...
 | /shrug *{message}*                                    | Add `¯\_(ツ)_/¯` to your message                                                                   | /shrug oh well                                             |
 | /status *{emoji_name}* *{descriptive status_message}* | Set a custom status that includes an optional emoji and a descriptive status message              | /status sick Feeling unwell and taking time off to recover |
-| /status clear                                         | Clear the current status                                                                          | /status clear                                              |
+| /status clear                                         | Clear the current status                                                                          | /status clear
+| /marketplace                                          | Open the Mattermost product Marketplace                                                           | /marketplace
```
Comment (justinegeffen) [inline site/content/integrate/slash-commands/built-in/_index.md:39]:
```suggestion
| /marketplace                                          | Open the Mattermost Marketplace                                                           | /marketplace
```

File: site/content/integrate/slash-commands/built-in/_index.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ Each Mattermost installation comes with some built-in slash commands that are re
...
-| /help                                                 | Open the Mattermost help page                                                                     | /help                                                      |
+| /help                                                 | Display Mattermost product documentation links                                                    | /help                                                      |
 | /invite *@{user}* *~{channel-name}*                   | Invite user to the channel                                                                        | /invite @john ~sampleChannel                               |
 | /invite_people *{name@domain.com ...}*                | Send an email invite to your Mattermost team                                                      | /invite_people john@example.com                            |
 | /kick (or /remove) *{@username}*                      | Remove a member from a Public or Private channel                                                  | /kick @alice                                               |
```
Comment (justinegeffen) [inline site/content/integrate/slash-commands/built-in/_index.md:25]:
```suggestion
| /kick (or /remove) *{@username}*                      | Remove a member from a public or private channel                                                  | /kick @alice                                               |
```

File: site/content/integrate/slash-commands/built-in/_index.md [hunk 5]
```diff
@@ -35,7 +35,8 @@ Each Mattermost installation comes with some built-in slash commands that are re
...
 | /shrug *{message}*                                    | Add `¯\_(ツ)_/¯` to your message                                                                   | /shrug oh well                                             |
 | /status *{emoji_name}* *{descriptive status_message}* | Set a custom status that includes an optional emoji and a descriptive status message              | /status sick Feeling unwell and taking time off to recover |
-| /status clear                                         | Clear the current status                                                                          | /status clear                                              |
+| /status clear                                         | Clear the current status                                                                          | /status clear
+| /marketplace                                          | Open the Mattermost product Marketplace                                                           | /marketplace
 
 {{<note "Note:">}}
 `/status` and `/status clear` slash commands listed above will be available in the Mattermost Mobile app in a future release.
```
Comment (justinegeffen) [inline site/content/integrate/slash-commands/built-in/_index.md:42]:
```suggestion
`/status` and `/status clear` slash commands listed above will be available in the Mattermost Mobile App in a future release.
```

**Review by justinegeffen — APPROVED** [review]
Just three corrections but otherwise LGTM!

File: site/content/integrate/slash-commands/built-in/_index.md (hunk 1 above)
Comment (justinegeffen) [reply site/content/integrate/slash-commands/built-in/_index.md]: Ah, I didn't see this before I reviewed, sorry! I'm not actually sure I know the difference between the two.

Comment (cwarnermm) [reply site/content/integrate/slash-commands/built-in/_index.md]: This slash command specifically opens the in-product Marketplace, and doesn't take users to the web-based Marketplace via https://mattermost.com/marketplace/. With this in mind, I'm going to leave the term "product" in the docs.

**Review by justinegeffen — APPROVED** [review]

**Review by hanzei — APPROVED** [review]

**Pull request merged**